running/SELinuxEscape/breakout.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+# breakout.sh demo script.
+# This script will demonstrate a lot of the features of podman, concentrating
+# on the security features.
+
+# Setting up some colors for helping read the demo output.
+# Comment out any of the below to turn off that color.
+bold=$(tput bold)
+blue=$(tput setaf 4)
+reset=$(tput sgr0)
+
+read_color() {
+    read -p "${bold}$1${reset}"
+}
+
+echo_color() {
+    echo "${blue}$1${reset}"
+}
+
```

Comment:
Hmm, I wonder if we could create a common include file for constants and functions like these.  I think so.

Reply:
But then again, one more thing to drag into place, maybe we shouldn't.